server/src/controllers/game.controller.js

```diff
@@ -100,6 +106,33 @@ class GameController {
       throw error;
     }
   }
+
+  async saveMoveToDb(req, res){
+
+    try{
+      //player_id and board_stte ideally would be passed from the request
+      const move = {player_id : 1, board_state: 2}
+
+      //no function has been created to save game to the database
+      //postman used to save sample data and here's the id
+      const gameId = "11"
+
+      // The update method currently provided by zuri core does not allow for direct addition into moves array
+      // So fetch the game, modify the moves then update the entire game
+      const game = await Game.fetchByGameId(gameId)
+      const gamePayload = game.data[0]
+      const moves = [...gamePayload.moves, move]
+      const newGamePayLoad = {game_id: gamePayload.game_id, moves}
+      
+      const object_id = gamePayload._id
+      await Game.update(object_id, newGamePayLoad)
+
+      res.status(201).json({success: true, message: "move saved succesfully"})
```

Comment:
Handling the request here means the initial `move` handler will throw an exception.